demos/speech_recognition_demo/python/utils/ctc_decoder_seq_pipeline.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2019-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+from copy import deepcopy
+
+from ctcdecode_numpy import SeqCtcLmDecoder
+
+from utils.alphabet import get_default_alphabet, load_alphabet, CtcdecoderAlphabet
+from utils.pipelines import BlockedSeqPipelineStage
+
+
+class CtcDecoderSeqPipelineStage(BlockedSeqPipelineStage):
+    def __init__(self, lm=None, profile=None, beam_width=500, max_candidates=None,
+            online=False):
+        assert profile is not None, "profile argument must be provided"
+        self.p = deepcopy(profile)
+        self.beam_width = beam_width
+        self.max_candidates = max_candidates
+        self.online = online
+
+        alphabet = self.p['alphabet']
+        if alphabet is None:
+            self.alphabet = get_default_alphabet()
+        elif isinstance(alphabet, str):
+            self.alphabet = load_alphabet(alphabet)  # shall not include <blank> token
```

Comment:
What is this case for? Default alphabet is always used. Plus you don't need a list of single chars as `self.alphabet`. It can be `str` (and `std::string`, I hope, for c++). As for the blank char `alphabet + ['']` in \_\_init__.py, it looks you can just leave it as `alphabet`.

Reply:
1. Removed default value for the alphabet. Now either filename, or a list.
2. Loading alphabet from a file because: a) we can read alphabet file in the same format as Mozilla DeepSpeech did, b) users may train their own model and provide its parameters in profile file.
3. Regarding blank symbol. There're actually two alphabets: one in CTC decoder (with blank), and another in LM (without blank). I think it was easier to provide each component with its correct alphabet, than to think where I should add 1, and where I shouldn't.